napari/_qt/_qapp_model/_tests/test_dock_widget_toggler.py

```diff
@@ -0,0 +1,25 @@
+from napari._app_model._app import get_app
+from napari._qt._qapp_model.qactions._window import ToggleDockWidgetAction
+from napari._qt.qt_main_window import Window
+from napari.components import ViewerModel
+
+
+def test_dock_widget_toggler():
```

Comment:
Have copied `napari/_app_model/_tests/test_viewer_toggler.py` - happy to take suggestions and to move location of test and `ToggleDockWidgetAction`.